internal/server/data/identity.go

```diff
@@ -374,7 +375,7 @@ type ListIdentityOptions struct {
 	LoadPublicKeys         bool
 }
 
-func ListIdentities(tx ReadTxn, opts ListIdentityOptions) ([]models.Identity, error) {
+func ListIdentities(c context.Context, tx ReadTxn, opts ListIdentityOptions) ([]models.Identity, error) {
```

Comment:
There's a `context.Context` in our transaction. We could expose that with a `Context()` method, but I think we probably don't need to be handling this in `data`. We can include the google provider in the API handler.

Reply:
Updated to not use the context after our discussion